src/main/java/io/pravega/perf/PerfStats.java

```diff
@@ -212,19 +212,7 @@ private long windowTimeMS(long time) {
         long totalLatency;
         long maxLatency;
         long totalBytes;
-        ArrayList<LatencyRange> latencyRanges;
```

Comment:
I'm not sure this change is needed.

Reply:
this is as part of code cleanup.. earlier i had class LatencyRange , now its replaced by Array.